packages/devtools_app/lib/src/logging/vm_service.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
i'd be in favor of renaming this file to `vm_service_logger.dart` to avoid any confusion with `vm_service/vm_service.dart`